packages/Python/lldbsuite/test/help/TestHelp.py

```diff
@@ -92,7 +92,7 @@ def test_help_version(self):
         version_str = self.version_number_string()
         match = re.match('[0-9]+', version_str)
         search_regexp = ['lldb( version|-' + (version_str if match else '[0-9]+') + '| \(swift-.*\)).*\n']
-        search_regexp[0] += '  Swift-\d+\.\d+'
+        search_regexp[0] += 'Swift version.*LLVM.*Clang.*Swift'
```

Comment:
This regexp only works on builds from source, not tagged builds.

Reply:
I assume I can just relax this test, it doesn't matter that much.